docs/conf/portland/2023/welcome-wagon.rst

```diff
@@ -3,5 +3,195 @@
+-  Starting Monday morning, check the unconference schedule in the Library/Astoria Room to see if there are any sessions you are interested in attending. New sessions are added all the time, so check back periodically.
+-  Eat! You can use the energy. There will be snacks in the Library/Astoria Room.
+-  Are you looking for a job or is there an opening at your company? Check out the job board and the job fair in the Library/Astoria Room.
+
+.. contents::
+   :local:
+   :backlinks: none
+
+Where is everything?
+~~~~~~~~~~~~~~~~~~~~
+
+-  The conference main stage and lightning talks are in Revolution Hall on the second floor.
+-  The unconference takes place on the first floor in the Library/Astoria Room.
+-  The Job Fair is on the first floor in the Library/Astoria Room.
+-  Snacks and drinks are on the first floor in the Library/Astoria Room.
+-  The quiet room is on the second floor in the Sunset Room.
+
+
+How should I get around?
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Portland is a very walk-able town.
+-  There are good public transportation options and taxi services. Check out the `Visiting Portland section of the Write the Docs website <https://www.writethedocs.org/conf/portland/2023/visiting/>`__ for more info.
+
+How should I dress?
+~~~~~~~~~~~~~~~~~~~
+
+-  Portland is a casual-dress town and so is the Write the Docs conference. You'll be meeting business colleagues at this conference, though, so neat and comfortable are good dress guidelines.
+-  If you are going on the `Write the Docs hike on Saturday <https://www.writethedocs.org/conf/portland/2023/outing/>`__, be sure
+   to bring appropriate hiking clothes and shoes. This time of year, the Pacific Northwest tends to be muddy or raining with occasional swaths of blue skies. Layering is usually the way to go.
+
+What will I eat?
+~~~~~~~~~~~~~~~~
+
+-  Snacks are provided for you on conference days.
+-  Coffee, tea, and water are always available in the Crystal Ballroom. Bring a water bottle to make it easier for you to stay hydrated.
+-  Food is available in restaurants in and around Revolution Hall. `Explore Portland's amazing food scene <https://www.writethedocs.org/conf/portland/2023/visiting/>`__. Invite someone you just met to join you! If you are invited to dinner, say yes! Making connections over dinner is a great way to get to know more people.
+
+Where should I sit?
+~~~~~~~~~~~~~~~~~~~
+
+-  The Revolution Hall main stage will have tables and rows of seats behind them.
+-  There's seating in the balcony that you can access from the third floor.
+-  There are no reserved seats; feel free to sit anywhere.
+-  If you can, show up early to the conference each morning to grab a seat at one of the round tables. Introducing yourself to your neighbors is one of the easiest way to meet people.
```

Comment:
just making sure this is still relevant for Rev Hall tables...

Reply:
Yea, we're working with them to get a few tables, but it won't be as many as in the Crystal for sure.